testdata/generate_test_data/create_complex_circular_references.py

```diff
@@ -181,12 +187,14 @@ def _make_inverted_complex_dependencies(replication_counter: str) -> list[etree.
     # E -> A / B / C (xml-text)
 
     all_resources = _make_list_of_resources(number_of_resources=5, replication_counter=replication_counter)
-    all_resources = _make_inverted_complex_dependencies_resource_ABC(all_resources)
+    all_resources = _make_single_link_inverted_complex_dependencies_resource_ABC(all_resources)
     all_resources = _make_inverted_complex_dependencies_resource_D(all_resources)
     return _make_inverted_complex_dependencies_resource_E(all_resources)
 
 
-def _make_inverted_complex_dependencies_resource_ABC(resource_list: list[etree._Element]) -> list[etree._Element]:
+def _make_single_link_inverted_complex_dependencies_resource_ABC(
+    resource_list: list[etree._Element],
```

Comment:
For 1 single parameter, I would also be okay to put it on 1 line (if the line is long enough)

Reply:
In this case the line was too long. Black added the trailing comma